AudioSensei/Views/MainWindow.axaml.cs

```diff
@@ -6,7 +6,9 @@
 using Avalonia;
 using Avalonia.Controls;
 using Avalonia.Input;
+using System.Windows;
```

Comment:
Have you ever heard about that little funny thing called alphabet?

Reply:
?